From: Karel Zak <kzak@redhat.com>
Date: Tue, 12 May 2009 19:22:19 +0000 (+0200)
Subject: blkid: make libuuid optional
X-Git-Url: http://git.kernel.org/?p=utils%2Futil-linux-ng%2Futil-linux-ng.git;a=commitdiff_plain;h=cdd17c7a73cab10a34bf80616225b8215ac4e2ed

blkid: make libuuid optional

read.c:28:23: error: uuid/uuid.h: No such file or directory
make[3]: *** [read.lo] Error 1
make[3]: *** Waiting for unfinished jobs....

Reported-by: Olaf <mailinglists@ban-solms.de>
Signed-off-by: Karel Zak <kzak@redhat.com>
---

diff --git a/libs/blkid/src/config.c b/libs/blkid/src/config.c
index 2e59218..9091736 100644
--- a/libs/blkid/src/config.c
+++ b/libs/blkid/src/config.c
@@ -24,9 +24,6 @@
 #include <errno.h>
 #endif
 #include <stdint.h>
-#ifdef HAVE_LIBUUID
-#include <uuid/uuid.h>
-#endif
 #include <stdarg.h>
 
 #include "blkdev.h"
diff --git a/libs/blkid/src/evaluate.c b/libs/blkid/src/evaluate.c
index a9c9fba..f0ae126 100644
--- a/libs/blkid/src/evaluate.c
+++ b/libs/blkid/src/evaluate.c
@@ -27,9 +27,6 @@
 #include <errno.h>
 #endif
 #include <stdint.h>
-#ifdef HAVE_LIBUUID
-#include <uuid/uuid.h>
-#endif
 #include <stdarg.h>
 
 #include "pathnames.h"
diff --git a/libs/blkid/src/read.c b/libs/blkid/src/read.c
index 4163874..b5e9cd0 100644
--- a/libs/blkid/src/read.c
+++ b/libs/blkid/src/read.c
@@ -25,7 +25,6 @@
 #endif
 
 #include "blkidP.h"
-#include "uuid/uuid.h"
 
 #ifdef HAVE_STRTOULL
 #define STRTOULL strtoull /* defined in stdlib.h if you try hard enough */
